torch_agents/cogment_verse_torch_agents/selfplay_td3/wrapper.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2021 AI Redefined Inc. <dev+cogment@ai-r.com>
```

Comment:
We have some of these functions already in `torch_agents/cogment_verse_torch_agents/wrapper.py`. Can they be used and move the additional ones to that file?